src/main/java/edu/kis/powp/jobs2d/drivers/DriversComposite.java

```diff
@@ -0,0 +1,49 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DriversComposite implements Job2dDriver {
+
+	private List<Job2dDriver> list;
+
+	public DriversComposite() {
+		this.list = new ArrayList<>();
+	}
+
+	public DriversComposite(List<Job2dDriver> list) {
+		this.list = list;
+	}
+
+	public void addDriver(Job2dDriver driver) {
+		this.list.add(driver);
+	}
+
+	public boolean removeDriver(Job2dDriver driver) {
+		return list.remove(driver);
+	}
+
+	@Override
+	public void setPosition(int x, int y) {
+		for (Job2dDriver driver : list) {
+			driver.setPosition(x, y);
+		}
+	}
+
+	@Override
+	public void operateTo(int x, int y) {
+		for (Job2dDriver driver : list) {
+			driver.operateTo(x, y);
+		}
+	}
+
+	public String toString() {
+		StringBuilder stringBuilder = new StringBuilder();
+		for (Job2dDriver driver : list) {
+			stringBuilder.append(driver.toString()).append("\n");
+		}
+		return stringBuilder.toString();
```

Comment:
I think that one-liner form would be more safe & useful, e.g. "Composite of _driverA_, _driverB_, _driverC_"